pass-data-client/src/test/java/org/eclipse/pass/support/client/JsonApiPassClientIT.java

```diff
@@ -154,19 +154,59 @@ public void testUpdateObject() throws IOException {
 
         // Try to update pub1 attributes
         pub1.setTitle("Different title");
+        client.updateObject(pub1);
+        assertEquals(pub1, client.getObject(pub1));
 
+        // Try to remove attribute
+        pub1.setTitle(null);
         client.updateObject(pub1);
         assertEquals(pub1, client.getObject(pub1));
 
         // Try to update sub attributes and relationship
         sub.setSource(Source.OTHER);
         sub.setSubmissionStatus(SubmissionStatus.CANCELLED);
         sub.setPublication(pub2);
+        client.updateObject(sub);
+        assertEquals(sub, client.getObject(sub, "publication"));
 
+        // Try to remove the relationship
+        sub.setPublication(null);
         client.updateObject(sub);
         assertEquals(sub, client.getObject(sub, "publication"));
     }
 
+    @Test
+    public void testUpdateObjectMultipleRelationships() throws IOException {
+        Repository rep1 = new Repository();
+        rep1.setDescription("one");
+
+        Repository rep2 = new Repository();
+        rep2.setDescription("two");
+
+        client.createObject(rep1);
+        client.createObject(rep2);
+
+        Submission sub = new Submission();
+        sub.setSubmitterName("Bob");
+
+        client.createObject(sub);
+
+        // Add multiple value relationship
+        sub.setRepositories(Arrays.asList(rep1, rep2));
+        client.updateObject(sub);
+        assertEquals(sub, client.getObject(sub, "repositories"));
+
+        // Delete one value of relationship
+        sub.setRepositories(Arrays.asList(rep2));
+        client.updateObject(sub);
+        assertEquals(sub, client.getObject(sub, "repositories"));
+
+        // Try to delete the relationship
+        sub.setRepositories(null);
+        client.updateObject(sub);
+        assertEquals(sub, client.getObject(sub, "repositories"));
```

Comment:
Should you add an assertion that check `sub.getRepositories` to verify it returns an empty list?